tests/conftest.py

```diff
@@ -226,6 +227,57 @@ def fixture(session):
 )
 
 
+def make_spectrum_source_fixture(info):
+    @pytest.fixture
+    def fixture(session):
+        sid = info["source_id"]
+        try:
+            session.create_spectrum_source(
+                sid, meta=info["meta"], channels=info["channels"]
+            )
+            for m in info["measurements"]:
+                session.create_spectrum_measurement(
+                    sid, m["freq_range"], m["timestamp"], m["data"], m["meta"]
+                )
+            yield info
+        finally:
+            session.delete_source(sid)
+
+    return fixture
+
+
+empty_spectrum_source = make_spectrum_source_fixture(
+    {
+        "source_id": uuid.uuid1().hex,
+        "meta": {"meta_test": "empty"},
+        "channels": ["ch"],
+        "measurements": [],
+    }
+)
+
+simple_spectrum_source = make_spectrum_source_fixture(
+    {
+        "source_id": uuid.uuid1().hex,
```

Comment:
Make use of a helper method like `get_random_source_id`. You can refer to `vibium-cloud` / `vibium-pipeline`